src/Graphics/Datatypes/RenderFieldState.h

```diff
@@ -26,12 +26,11 @@
 */
 
 
-#ifndef CORE_ALGORITHMS_VISUALIZATION_RENDER_FIELD_STATE_H
-#define CORE_ALGORITHMS_VISUALIZATION_RENDER_FIELD_STATE_H
+#ifndef GRAPHICS_DATATYPES_RENDER_FIELD_STATE_H
+#define GRAPHICS_DATATYPES_RENDER_FIELD_STATE_H
 
 #include <Core/Datatypes/Color.h>
-#include <array>
-#include <Core/Algorithms/Visualization/share.h>
+#include <Graphics/Datatypes/share.h>
```

Comment:
```suggestion
#include <array>
#include <Graphics/Datatypes/share.h>
```

Also needed for Windows